tests/qt/test_file_path_options.py

```diff
@@ -0,0 +1,119 @@
+import os
+import pathlib
+from unittest.mock import patch
+
+import pytest
+from PySide6.QtGui import (
+    QAction,
+)
+from PySide6.QtWidgets import QMenu, QMenuBar
+
+from tagstudio.core.enums import SettingItems
+from tagstudio.core.library.alchemy.library import LibraryStatus
+from tagstudio.qt.modals.settings_panel import SettingsPanel
+from tagstudio.qt.widgets.preview_panel import PreviewPanel
+
+
+# Tests to see if the file path setting is applied correctly
+@pytest.mark.parametrize(
+    "filepath_option", ["show full path", "show relative path", "show only file name"]
```

Comment:
*(In the context of enums)* This is also how the tests would look when using enums instead of the old strings. 
```suggestion
    "filepath_option",
    [
        ShowFilepathOption.SHOW_FULL_PATHS.value,
        ShowFilepathOption.SHOW_RELATIVE_PATHS.value,
        ShowFilepathOption.SHOW_FILENAMES_ONLY.value,
    ],
```